airgun/entities/rhsso_login.py

```diff
@@ -25,6 +26,12 @@ def logout(self):
         view = RhssoExternalLogoutView(self.browser)
         return view.read()
 
+    def get_two_factor_login_code(self, values, url):
+        self.browser.selenium.get(url)
+        self.login(values)
+        view = RhssoTwoFactorSuccessView(self.browser)
```

Comment:
Could you explain what exactly is happening here?

Why does test need to pass URL? What this URL is?
`self.login` will change view for us. And then we say we are on specific view, anyway?

Reply:
I missed adding the description, Yes the URL different from satellite url, and there is no navigation to that url from the satellite (mentioned in the description). Yes, we have a new view after login.

Reply:
Thanks, makes sense now.